### PR TITLE
multi-tenant support in firebaseui

### PR DESCRIPTION
**Pull request by wti806**
Added support for multi-tenant in FirebaseUI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1449,6 +1449,40 @@ ui.start('#firebaseui-auth-container', {
 });
 ```
 
+### Multi-tenancy support
+
+For [GCIP](https://cloud.google.com/identity-cp) customers, you can build a
```
Comment (bojeil-google) [inline README.md]: https://cloud.google.com/identity-platform

File: changelog.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+feature - Added supported for multi-tenancy for Google Cloud's Identity Platform developers.
```
Comment (bojeil-google) [inline changelog.txt]: Added multi-tenancy support for Google Cloud's Identity Platform developers.

Comment (wti806) [reply changelog.txt]: done

File: README.md (hunk 1 above)
Comment (wti806) [reply README.md]: done

**Pull request merged**